src/main/java/SkyPro/ExaminerApp/Controllers/JavaQuestionController.java

```diff
@@ -0,0 +1,35 @@
+package SkyPro.ExaminerApp.Controllers;
+
+
+import org.springframework.web.bind.annotation.*;
+import skypro.examinerservice.domain.Question;
+import skypro.examinerservice.service.JavaQuestionService;
+
+import java.util.Collection;
+
+@RestController
+@RequestMapping("/exam/java")
+public class JavaQuestionController {
+
+    private final JavaQuestionService javaQuestionService;
+
+    public JavaQuestionController(JavaQuestionService javaQuestionService) {
```

Comment:
инъекцию нужно делать по интерфейсу, а не реализации

```suggestion
    public JavaQuestionController(QuestionService javaQuestionService) {
```